frontend/src/hooks/useClearSWRCache.ts

```diff
@@ -1,14 +1,38 @@
-import { useSWRConfig } from 'swr';
+import { type State, useSWRConfig } from 'swr';
+
+interface Cache<Data = any> {
+    keys(): IterableIterator<string>;
+    get(key: string): State<Data> | undefined;
+    set(key: string, value: State<Data>): void;
+    delete(key: string): void;
+}
+
+export const manageCacheEntries = (
```

Comment:
what are you managing about the cache? manage is a very generic name that is not intention revealing

Reply:
is it clearCacheEntries?

Reply:
Yes